dpdk/vr_dpdk_lcore.c

```diff
@@ -266,6 +301,92 @@ vr_dpdk_lcore_if_schedule(struct vr_interface *vif, unsigned least_used_id,
     return 0;
 }
 
+/* Post an lcore command */
+void
+vr_dpdk_lcore_cmd_post(struct vr_dpdk_lcore *lcore, uint16_t cmd,
+    uint32_t cmd_param)
+{
+    uint64_t event = 1;
+
+    rte_atomic32_set(&lcore->lcore_cmd_param, cmd_param);
+    rte_atomic16_set(&lcore->lcore_cmd, cmd);
+
+    /* TODO: wake up the pkt0 thread? */
```

Comment:
Why TODO comment above - doesn't the vr_usocket_write() call below wake up pkt0 thread?